ci/templates/setup-tools-steps.yaml

```diff
@@ -3,7 +3,8 @@ steps:
   displayName: 'Setup Python'
   inputs:
     versionSpec: 3.11
-- task: UseRubyVersion@0
-  displayName: 'Setup Ruby'
-  inputs:
-    versionSpec: 3.1.x
+# Ruby is only needed as a dependency for UAAC
+# - task: UseRubyVersion@0
+#   displayName: 'Setup Ruby'
+#   inputs:
+#     versionSpec: 3.1.x
```

Comment:
Assuming this change is unrelated to this PR, was this tested with the UAA sample?
Should the code be deleted instead of commented out?

Reply:
This task caused _all_ of the pipelines to fail due to a change in the software on the DevOps agents, but is only potentially relevant to the SSO sample, which apparently doesn't even have a pipeline definition anymore.
I'd prefer to leave it like this until #274 is sorted out, at which time we can more properly evaluate whether UAAC is needed at all for that integration test.